scripts/test/cleanup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+docker compose -f "$(dirname $0)/../../dev-compose.yaml" down
```

Comment:
isn't the `down -v` better? I know that we don't use default docker volume folder location, but anyway it might do some additional cleanup...

Reply:
Done.